third_party/glad/README.md

```diff
@@ -0,0 +1,25 @@
+# Introduction
+
+The `GLAD` library is used for the texture of glfw, which is used to create textures to render RGBA data.
+
+# Notice
+
+The `glad` source is generated from https://gen.gald.sh.
+
+Step:
+
+1, Add APIs:
+
+gl : `Version 3.3`
+
+2, Select Extensions:
+* `GL_ARB_multisample`
+* `GL_ARB_robustness`
+* `GL_KHR_debug`
```

Comment:
To confirm: your texture patch requires all of these extensions?

Reply:
Well, I tested it again, no need to add any extensions, so remove it.